include/seastar/core/sstring.hh

```diff
@@ -781,9 +781,14 @@ std::ostream& operator<<(std::ostream& os, const std::unordered_map<Key, T, Hash
 }
 }
 
-#if FMT_VERSION >= 90000
+#if FMT_VERSION >= 90000 && !defined(FMT_DEPRECATED_OSTREAM)
```

Comment:
Why is this needed? Doesn't the formatter take precedence when FMT_DEPRECATED_OSTREAM is defined?

Reply:
> Why is this needed? 

no, it's not needed. i was misled by an error which was not relevant. 

> Doesn't the formatter take precedence when FMT_DEPRECATED_OSTREAM is defined?

agreed. reverted this change.